specification/batch/Azure.Batch/tspconfig.yaml

```diff
@@ -0,0 +1,30 @@
+parameters:
+  "service-dir":
+    default: "sdk/batch"
+emit:
+  - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "./data-plane"
+    emitter-output-dir: "{project-root}/.."
+    examples-directory: "{project-root}/examples"
+    output-file: "{azure-resource-provider-folder}/{service-name}/preview/{version}/BatchService.json"
```

Comment:
Since we switching to track2 with the adoption of TypeSpec our namespace is changing to Azure.Batch as part of that.   We will have an overlap of releasing both track1 and track2 for a time period as we don't expect all of our customers to switch over immediately.  Also, as part of track2 we have changed a number of our model names so we need to keep two versions around till we can get our customers off of track1.

Reply:
The TypeSpec and Swagger here is independent of track1/track2 SDK. There were track2 SDK GAed from Swagger (now this is no longer allowed).

Here TypeSpec/Swagger is only about the REST API of your service, regardless of client.

Let us have opinion from Wes @weshaggard 
Question:

Do we allow some Swagger under `data-plane/Microsoft.Batch`
https://github.com/Azure/azure-rest-api-specs/tree/main/specification/batch/data-plane/Microsoft.Batch
and some other Swagger under `data-plane/Azure.Batch`
https://github.com/Azure/azure-rest-api-specs/tree/abf00209016bb14051444310dc38d6602ad03ea0/specification/batch/data-plane/Azure.Batch

Or do we prefer having them under same folder.

They are same service and same REST API (difference is the api-version, and the one under `data-plane/Azure.Batch` is from TypeSpec).

Reply:
I would say we should keep them under the same RP name. @allenjzhang may have more knowledge here but I don't think we have been changing any other paths to be Azure.* in the specs repo.